next.config.js

```diff
@@ -2,6 +2,11 @@
 const nextConfig = {
   reactStrictMode: true,
   swcMinify: true,
-}
+};
 
-module.exports = nextConfig
+module.exports = nextConfig;
+module.exports = {
+  images: {
+    domains: ["images-na.ssl-images-amazon.com"],
+  },
+};
```

Comment:
Is this the expected? Because L8 will `override` L7. How about:

```suggestion
module.exports = {
   ...nextConfig,
  images: {
    domains: ["images-na.ssl-images-amazon.com"],
  },
};
```